samples/nrf9160/modem_shell/sample.yaml

```diff
@@ -162,7 +162,7 @@ tests:
     build_only: true
     integration_platforms:
       - nrf9160dk_nrf9160_ns
-    platform_allow: nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
     extra_args: >
       OVERLAY_CONFIG="overlay-modem-trace.conf;overlay-memfault.conf"
       DTC_OVERLAY_FILE="nrf9160dk_ext_flash.overlay"
```

Comment:
nRF9161 DK build should probably use `nrf9161dk_ext_flash.overlay` instead.

Reply:
I'm not sure what is the best way to allow different overlays based on the platform in this case, so I'll drop this for now and add it later for this test.